src/CourseDashboard.hpp

```diff
@@ -1,7 +1,19 @@
 #pragma once
+#include "User.hpp"
+#include <vector>
+
 
 class CourseDashboard
 {
 public:
     CourseDashboard();
+    void showAll();
+    void createUser(const User & user);
+    void deleteUser(const User & user);
+    void retriveUser(const User & user);
```

Comment:
I would expect that function named retrieveUser will return a User. Here I need to provide it with a User. How can I get the user that I need to pass here?

Reply:
Fixed